bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/group-thing-types.xml

```diff
@@ -30,7 +30,6 @@
 		<description>"On" if all lights in this group are "On", otherwise "Off".</description>
 		<tags>
 			<tag>Status</tag>
```

Comment:
Is it on purpose that Status is left for **all_on**, but removed for **any_on**?

Reply:
Yes, my idea is that it could make sense to have as status for the group that all lights are ON or not.
But now, I told me that I could rather use Measurement + Light. Should I put them on all_on or any_on ?
Finally, the more appropriate could be Measurement + Light only on any_on ?

Reply:
In fact, this is the same case as "is_raining" on another binding. As discussed in #12262, I finally tagged with Measurement and Rain. So, I believe I should use here Measurement + Light. I am just not sure on which of both channel types.

Reply:
These channels are for `lightgroup`. I don't think they represent any kind of measurement, but simply logic. @J-N-K, please correct me if I'm wrong.

Reply:
They are read-only channels type. The tags could help to know if this group is on or off.

Reply:
I pushed the change as it is more coherent with what was proposed in #12262

Reply:
My point is that they represent logic, not measurements. "Measurement" + "Light" seems to indicate that it's a measurement of light?

Reply:
Measurement if there is light or not light.
Maybe @kaikreuzer can comment as it looks to me relatively similar to the case "is raining" which was discussed earlier.

Reply:
I'm with @jlaur here - in this case, we don't have a sensor that measures anything, but it is a channel which reports a (logical) state of the lights. Maybe the light is on, but the bulb is broken, so it is still dark ;-) A sensor would notice, a logical state reporting channel doesn't.
Especially on such grouping features, I'd remove the tags completely.

Reply:
I think in that scenario ("is raining") it was based on sensor data, i.e. an actual measurement, just in low resolution (boolean).

Reply:
Ok guys, I remove it.